docs/Hardlinks/Hardlinks-and-Instant-Moves.md

```diff
@@ -4,18 +4,18 @@
 
 !!! info
 
-    If you’re wondering why hard links aren’t working or why a simple move is taking far longer than it should.
+    If you’re wondering why hardlinks aren’t working or why a simple move is taking far longer than it should.
 
     Here we will try to explain it.
 
-This Guide exist out of 4 Sections
+This Guide exists out of 4 Sections
 
 1. This page with a short description.
-1. [How to set up for](/Hardlinks/How-to-setup-for/) your installation method.
-1. [Examples](/Hardlinks/Examples/) what you should use for your path settings in your used applications.
+1. [How to set up for](/Hardlinks/How-to-setup-for/) Your installation method.
+1. [Examples](/Hardlinks/Examples/) What you should use for your path settings in your used applications.
```

Comment:
```suggestion
1. [Examples](/Hardlinks/Examples/) what you should use for your path settings in your used applications.
```

Reply:
Funny that grammarly told me it should be changed to capitol W

Reply:
That's because grammarly doesnt understand markdown. You have to be aware that punctuation typically marks the end of a sentence, and markdown can fuck that up.